Admin/BaseMediaAdmin.php

```diff
@@ -106,7 +106,7 @@ public function getNewInstance()
 
         if ($this->hasRequest()) {
             if ($this->getRequest()->isMethod('POST')) {
-                $media->setProviderName($this->getRequest()->get(sprintf('%s[providerName]', $this->getUniqid()), null, true));
+                $media->setProviderName($this->getRequest()->get(sprintf('%s', $this->getUniqid()), null, true)['providerName']);
```

Comment:
You can use `version_compare` and [this constant](https://github.com/symfony/http-kernel/blob/2.8/Kernel.php#L62) to achieve that. @sonata-project/contributors : since the constant is part of http-kernel, it's ok.